site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -217,7 +229,7 @@ through [Github issues](https://github.com/googleforgames/agones/issues).
 ```bash
 gcloud container node-pools create windows \
   --cluster=[CLUSTER_NAME] \
-  --no-enable-autoupgrade \
+  --region=[COMPUTE_REGION] \
   --image-type WINDOWS_LTSC_CONTAINERD \
   --machine-type e2-standard-4 \
   --num-nodes=4
```

Comment:
This one should also say "where `[CLUSTER_NAME]`is the name of the cluster you created and `[COMPUTE_REGION]` is the region."

Reply:
This is getting really redundant, should we just have a tutorial on creating a node pool and stamp them out in different ways?

Reply:
I agree it's redundant. We could make a template but I think that might be more work to maintain than the 3 copies of the docs we currently have.